data/mods/Aftershock/items/afs_armor.json

```diff
@@ -276,5 +276,23 @@
       "flags": [ "MAG_COMPACT" ]
     },
     "flags": [ "STURDY", "OUTER" ]
+  },
+  {
+    "id": "afs_sentinel_cloak",
```

Comment:
I thought afs prefix was going away?

Reply:
I mean the afs prefix used in the actual filenames AKA afs_armor.json in this case.